src/djlint/rules.yaml

```diff
@@ -172,7 +172,7 @@
     message: Do not use entity references.
     flags: re.I
     patterns:
-    - '&.{5};'
+    - '&(?!(lt|gt|amp|quot|nbsp|ensp|emsp|thinsp))[^;]+;'
```

Comment:
I'm guessing you'd want this list of entities to be a config setting. Happy to do that